config/gregtech/worldgen/overworld/monazite_vein.json

```diff
@@ -0,0 +1,37 @@
+{
+    "weight": 30,
+    "density": 0.4,
+    "max_height": 40,
+    "min_height": 20,
+    "surface_stone_material": "monazite",
+    "generator": {
+        "type": "ellipsoid",
+        "radius": [
+            13,
+            13
+        ]
+    },
+    "filler": {
+        "type": "simple",
+        "value": {
+            "type": "weight_random",
+            "values": [
+                {
+                    "weight": 60,
+                    "value": "ore:bastnasite"
+                },
+                {
+                    "weight": 20,
+                    "value": "ore:monazite"
+                },
+                {
+                    "weight": 20,
+                    "value": "ore:neodymium"
+                }
+            ]
+        }
+    },
+    "dimension_filter": [
+        "dimension_id:1"
```

Comment:
This is fine to leave in the end

Reply:
i moved it back to ow, seems uselses to me